src/main/java/net/teamterminus/machineessentials/energy/electric/template/ElectricDeviceBlockEntity.java

```diff
@@ -72,29 +72,29 @@ public long receiveEnergy(@NotNull Direction dir, long amperage) {
                         boolean pathBroken = false;
                         //handle wires with insufficient voltage rating
                         for (NetworkComponent pathBlockEntity : path.path) {
-                            if(pathBlockEntity instanceof ElectricWire pathWire){
-                                if(pathWire.getVoltageRating() < voltage){
+                            if (pathBlockEntity instanceof ElectricWire pathWire){
+                                if (pathWire.getVoltageRating() < voltage){
                                     pathWire.onOvervoltage(voltage);
                                     pathBroken = true;
                                     pathVoltage = Math.min(pathWire.getVoltageRating(), pathVoltage);
                                     break;
                                 }
                             }
                         }
-                        if(pathBroken) continue;
+                        if (pathBroken) continue;
 
-                        if(pathVoltage > 0){
+                        if (pathVoltage > 0){
                             //handle device over-voltage
-                            if(pathVoltage > getMaxInputVoltage()){
+                            if (pathVoltage > getMaxInputVoltage()){
```

Comment:
I love that there's so many of them. Gotta have so much fun writing these comments